indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -148,6 +151,8 @@
                 confirmed = confirmPrompt(promptMsg, $(this).data('title') || $T('Confirm action'));
             }
             confirmed.then(execute);
+        }).on('click', selectors.join('.disabled, ') + '.disabled', function(e) {
```

Comment:
actually, now that i see this: why not leave the selectors unchanged and simply do `if ($this.hasClass('disabled')) { return; }` after the initial preventDefault?